src/updateFileFromServer.ts

```diff
@@ -60,11 +60,60 @@ export async function toggleServerTaskStatus(e: Editor, settings: TodoistSetting
 		const serverTaskName = (await api.getTask(taskId)).content;
 		if (tryingToClose) {
 			await api.closeTask(taskId);
-			new Notice(`Todoist Text: Closed "${serverTaskName}" on Todoist`);
+			
+			const actionedTaskTabCount = lineText.split(/[^\t]/)[0].length;
```

Comment:
The current code doesn't handle when tasks are re-shuffled. The other approach we could take to avoid this would be to do a search within the file for a line matching each of the known subtask URLs. 


I think the code here is mergeable though - if someone has an issue with re-shuffling lines on their file causing unexpected behavior, they can file an issue and we can think about it later

Reply:
Oh, you're absolutely right, as I definitely didn't consider that scenario.  I was quite happy with figuring out how to traverse the editor API and didn't think of the more expansive options.  I think you're right to proceed now.  It feels almost like an edge case of an edge case; definitely possible, but not something I suspect most users would run into.